update-docs-sources

```diff
@@ -13,7 +13,8 @@ done
 langs=`/usr/bin/php -r 'include "/local/src/phpweb/include/languages.inc"; echo implode( " ", array_keys( $ACTIVE_ONLINE_LANGUAGES ) );'`;
 langs="base $langs"
 for i in $langs; do
-  cd /local/src/phpdoc-git/$i
+  lang=`echo "$i" | tr '[:upper:]' '[:lower:]'`
+  cd /local/src/phpdoc-git/$lang
```

Comment:
Is a similar fix needed for the production build?

Reply:
As the production build doesn't updatre the docs-sources by itself I doubt it. I'll recheck though!

Reply:
And the [cronjob on rsync.php.net](https://github.com/heiglandreas/systems/blob/patch-2/rsync.php.net/cronjobs.docbuilder) runs a file `/home/docbuilder/build-docs.sh` which I can't find in this repo. So perhaps someone with access to that box (@derickr ?) needs to have a look for that...

Reply:
That file got dropped in 66433261d203d4865ff34f6f9d8d6d59a50665b5. As far as I can see, build-docs-rsync is the production build.

Reply:
Fixed the same issue in `build-docs-rsync`.